packages/testcontainers/src/generic-container/generic-container-builder.ts

```diff
@@ -44,6 +48,12 @@ export class GenericContainerBuilder {
     return this;
   }
 
+  /**
```

Comment:
I really really really really appreciate these comments for our methods and functions but, since they are not anywhere in the library, I don't know if it would be convenient to start doing it. 🤔